src/components/EmbedMap.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect } from "react";
+import { Map, GoogleApiWrapper, Marker } from "google-maps-react";
+import { connect } from 'react-redux'
+
+const EmbedMap = props => {
+  const { lat, lng } = props.coords
+
+  const onClickHandler = e => {
+    props.changeCoords({ lat: e.latLng.lat(), lng: e.latLng.lng() });
+  };
+
+  const getCurrentLocation = () => {
+    if (window.navigator && window.navigator.geolocation) {
```

Comment:
maybe a redundant question, but what happens if the user does not want to share their location?

Reply:
state for coords will be initialState (0, 0), map will center on Google Maps default location (San Franscisco).
Could choose our own position to default too (lat 0, long 0 is in the sea so probably don't want that)

Reply:
ok cool